README.rst

```diff
@@ -17,7 +17,7 @@ Install:
 
 .. code:: bash
 
-    $ curl -L https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash
+    $ curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
```

Comment:
I confirmed that the new URL is equivalent (=~ GH returns 302) as of Feb 27 2018.

```
% curl -Is https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | grep -i Location:
Location: https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer
```

// In #4, we've switched the download URL from raw.github.com to raw.githubusercontent.com. This is 2nd time to switch the URL.